runner/ansible/README.md

```diff
@@ -8,17 +8,35 @@ To be defined...
 
 ## Metadata files
 
-The metadata files provide information about the test themselves. They are used to get information
+The metadata files provide information about the check themselves. They are used to get information
 from the Trento Web GUI and render properly everything related with the Ansible tasks.
 
 In order to use them properly, some fields are required. An example is available at [defaults/main.yml](roles/checks/1.1.1/defaults/main.yml).
 
 These are the fields needed by Trento:
 
-- `id`: The test unique identifier.
-- `name`: A short name for the check. It is used as more user friendly identifier for the test
+- `external_id`: The check unique identifier. This value must not be changed during the lifetime of the check
+- `id`: The internal identifier. It is used to identify the checks internally (a more user-friendly nomenclature), to specify the expected values and to apply the ordering to the lists in the web side. This value can be changed to adapt to any new internal need.
```

Comment:
I am very confused by this. Why are we keeping the old id? Having two ids seems very complicated to me. My expectation is that we would just use the new alfanumeric convention in the existing `id` field.

Reply:
Sorry, rushed to review without reading the opening post. 😅 

So, I really don't like having two id fields: it makes things very confusing.

If the purpose of what was previously called `id` is now just to order things and being a cosmetic name, then I would rename to something else. The identifier, as a concept, should be just one; here we instead are telling advanced users that they need to come up with two ways to identify the checks, which will be very confusing and cumbersome to them.

Reply:
We already have a `name` field to provide a user-friendly identifier, so I guess the only thing you want here is an optional `order` field.

Reply:
I knew that we would have this discussion hehe
Agree that having 2 ids is confusing (unless you read the documentation).

Let's do the next:
- `external_id` is renamed as `id`, so we have one id field
- The current `id` is renamed to `name`. We remove the current `name` content (we don't use it anywhere). The `name` is a developer friendly string, which is actually used to specify the expected values map, etc
- New `order` field to specify the check order **within** the test group (so we don't need to change the order for all the checks if we only want to change the check order withing the group)